src/compiler/resolutionCache.ts

```diff
@@ -769,26 +784,28 @@ export function createResolutionCache(resolutionHost: ResolutionCacheHost, rootD
                                     Diagnostics.Reusing_resolution_of_module_0_from_1_of_old_program_it_was_successfully_resolved_to_2 :
                                 Diagnostics.Reusing_resolution_of_module_0_from_1_of_old_program_it_was_not_resolved :
                             resolved?.resolvedFileName ?
-                                resolved.packageId ?
-                                    Diagnostics.Reusing_resolution_of_type_reference_directive_0_from_1_of_old_program_it_was_successfully_resolved_to_2_with_Package_ID_3 :
-                                    Diagnostics.Reusing_resolution_of_type_reference_directive_0_from_1_of_old_program_it_was_successfully_resolved_to_2 :
-                                Diagnostics.Reusing_resolution_of_type_reference_directive_0_from_1_of_old_program_it_was_not_resolved,
+                            resolved.packageId ?
+                                Diagnostics.Reusing_resolution_of_type_reference_directive_0_from_1_of_old_program_it_was_successfully_resolved_to_2_with_Package_ID_3 :

```

Comment:
Another inoccrect indentation after nested ternaries ?

Reply:
Honestly I'm not sure; this is so nested and deep that I can't tell what's right.

looking at this it kinda makes sense? https://dprint.dev/playground/#code/C4JwhgxgpgFAsAKAAQqQCwPYGdgBpGpIAOUIAYgJYA2UAwpGlEgLytIhRYZUBuUAJgFkM-AK40AcmAC2nJGCxJRAOwDWyjAHdlSAPwFCqDl14DdAOmPc+-SpJlN9yQy6un+5opFVgA5lABJfj0DFzD5JAAuUPDDACMomPCIROdXTmszSwz3OygpWRC02LcbT28-QOCnWLDg6OLalCYGpqQAM3xi5QcuwggMZWAwCmVR3zy+oxybC1KBPIKoKZR5-gsvCB9-IKQAMj3iCp3+ABUMAGVQcZg18q3KoIBKRCeAbiA/config/N4WAUABF1QRAlgOwCYFNEBcDq9kYBawBcEALADTgywA2SqOehJAjAAweWTSyKoDuAGXoBpJMmIRYAQwCuGAPawu1WQGdUAFWkAjNZIBm0mhpUxzsDQFt4AYQU0FifSRk1+0gJ76z5qQEdZBQxUAGUMTxpUSVgABwAnVANUeIARBVkdKOUqPwCgkIAFeIVYlykAYyc1eDUQzBzuP1h1VAAheOkK1HLYfnx0ADlg0KQAcyjhPka8qR1O7sKFGox4Jxi1aStUKdQAVUQotTUACWlEMfGZvMtxqLaFZE8llbXEDa2d+mvmvgAPDD2TAlGgAMUc-Be8FW61c-wwux+FgwnXgdAu9isVmkvScNE8AFlZDRVojfBYEkkUmcLlcSEYTKhydRSilpIp4lCYe9XNikBhpEhGrlmtJ4iV+KDZIgKtyAHStQpi9AEHo9GKU5LxYbTZk8KooaFvYwAUT+lOObzl6NQhRSZotNVhEAZphFFgM0tlbwdiUtTjlaliXVQAEEDCF4lKZdyRKhPPwFPEJCQUbIme7qPArLEkxhUqgKjQxeyrQYk90iST4IjU-F03q4Aa6vXZUnNJ5YqhA8HuuHI4MBHGE0mUxA0xmmhZm2nZaMxoh2bJEj2Q-2UoP+MPE8nJBOuJmeFZHsTu2o8wBJHN5gtFkvc3p8zCC96NqTH5CnwN5h03wvFzoHxiJ8BSFN9YFQc0-zvQCrXPeIMEGT5kF-BDHxfUDX0POBs1zBDbwA0sA3gxDkKvPCMHQ-kX1gcAAF9wCAA/language/typescript

Reply:
I think there is issue with dprint here.. 
https://dprint.dev/playground/#code/PQKhFgCgAIWhxApgFwM7WQC0dADgJwHsBzfAQwFtoBLAOwDND8KzlrDaNDoAjHRCtWTJEAExqceAV2oAbUQEkG3VNyytoAdxwBjMpymocQrtB35ErHLUSa8RUpSixgUelNo62HaMRQAhGXklRgAKVGQrAC5oQLlRRHwABQdyCgBlSJEAGl4PUVlEGMDaAsQ44OVoAB9ofMR6OjEASmKgxSqAb2doXp0OCLMpfAtaZAARagsvJgBPaABeaHHEaWIAOh1sHQBrFcabUXCsxHWCEjTm9b9kAGFh0YmpxBn8WdDmgG4evoHkPPiIUIk2myDmi18KBBLzBbySrEwoRuADkmCxZNQAF5iACCPFUsikInhWCRKAAKqgKh1GABRQTIADyRNwRIAYnJECTERErJtCBRcJz8IzcN5aKhmgBCXI6B6IMbQ16zZpfH7QYDAaC3DgAN0S-ywOEahWgtEoODB0G50GwFho9C0OCMhuw0BNiGRFokEUs4ithDF1EE2OgFCksjYuFNonWaHdnOgAiE6H04kwZF1dGIpn6gqJOFu+g41D0sjwCPV-Ql-1kVgitwztD8onGaA5pqWvJE6zrIgbTZbbdQHZwAH5oBY+9R9eTCNSgbSJcNs3iCQXucc+X3EAP9EP25zmtAYvUDmJvjBemY-gnCl6KLuYhF8NmANoAXQhn8v15vNbvT0LTnUcFHEJYbDsABZMhcAAHhfbNchSC5KAXZRQNEAA+D5f2vatBiIQhkBiFDHAodDGAAJUIYjP2-D88N6ahHVCQh2U5LcezzIVChFIMBlVaBuivP9ek1bVA3mMhZHLD0gVyfV8FQdhODTGhBQxMQ2TRVh1T-CScXoegYXQYhZEIHgZOgVB+lwHB1JU4hzWQYYcFEQhdzGMNWBEfBeBeMhDBwTQhEwaAAFoIvY11EGkx4zBkuUd3EJgzUQMQxFMBI-MEGw6jFbhRCsdLUFQfT8NveTlHQJYyBGMhZjZIgKC405qsYVB1gVZBX13D5clCN8djixSZKkRAP1aa1UjQ-JCnpIRKMIUCqgWLDhIqsSNS1JdUDcwCwK2sSCP+eTwK4TDQhGlUmO269etmHFRFEGjiOu0bDtENVRPu3oLFc-BOF1cbTmDaNqG03T-jHY6-s6aAlJUjgYhBwlTiR1TcnBrTRB05hWFR0H1hxyG8eh3InJctzT1KBomlEXIyGM0z4AsqzZHSOyijqOnz3EABfE84futGJvWTGODuv8BZ+v7TvsVCKFI2aKPmxBFuQZaIREv6-w9B9d2yEXeg6whUGN36-qI5BLb169A3FVAYmrJTkDnMi0mWnVBWFUUnba-lfb4-3VMlO37egGL0moZzWBpmyTnWaPY+piwI-tnc92bMRh1HDO9fs0ps01mIpW7U5znIzWkgVURs2gWGrcjs8GZPcStVr4vmxodBW8OKP-NRfLqHQBV+gScQyD7vmGZNsSK7OVWa7rhuFnXyEtfaRJR01wOeL9gSJWPJvI71+gZKMduJK7+ue+Tf5R5s71p+gHLEmTAeeHmA+Q6P8rm5n16IvKuaQV7d2ILkQyXgpDWTFogY6AtpbXgkj7aMKBLRumkKUU0dBGAE3FA6J00B6ruUIA3XA9U2DWQftANitBZDzAAFboDShwRhb80DNGOgrBGLDciiDQLKAUFAODpHYvgHQiAlSwlmJTCRUjRzoCFksbBZQpTIOAUnNRnY8g4JwEsXWkc8yiNoOI4YUiZFzALn9VQFjEBKJsfdFhMQWE1FqKEcuSdf6JFDgMbqDIVg6DrOQcUjIGHzAnIvHRiAxzrDcbTBI-NmhOO2oI52nD0DVA8TcTWQSQmsDDvvAUvFfH-xPonPkMS4npPbv3FoqToBIOOgDYYnBzCWBEMtUIoDKC5BiXLa8AsoBCyAdtIAA/config/N4WAUABF1QRAlgOwCYFNEBcDq9kYBawBcEALADTgywA2SqOehJAjAAweWTSyKoDuAGXoBpJMmIRYAQwCuGAPawu1WQGdUAFWkAjNZIBm0mhpUxzsDQFt4AYQU0FifSRk1+0gJ76z5qQEdZBQxUAGUMTxpUSVgABwAnVANUeIARBVkdKOUqPwCgkIAFeIVYlykAYyc1eDUQzBzuP1h1VAAheOkK1HLYfnx0ADlg0KQAcyjhPka8qR1O7sKFGox4Jxi1aStUKdQAVUQotTUACWlEMfGZvMtxqLaFZE8llbXEDa2d+mvmvgAPDD2TAlGgAMUc-Be8FW61c-wwux+FgwnXgdAu9isVmkvScNE8AFlZDRVojfBYEkkUmcLlcSEYTKhydRSilpIp4lCYe9XNikBhpEhGrlmtJ4iV+KDZIgKtyAHStQpi9AEHo9GKU5LxYbTZk8KooaFvYwAUT+lOObzl6NQhRSZotNVhEAZphFFgM0tlbwdiUtTjlaliXVQAEEDCF4lKZdyRKhPPwFPEJCQUbIme7qPArLEkxhUqgKjQxeyrQYk90iST4IjU-F03q4Aa6vXZUnNJ5YqhA8HuuHI4MBHGE0mUxA0xmmhZm2nZaMxoh2bJEj2Q-2UoP+MPE8nJBOuJmeFZHsTu2o8wBJHN5gtFkvc3p8zCC96NqTH5CnwN5h03wvFzoHxiJ8BSFN9YFQc0-zvQCrXPeIMEGT5kF-BDHxfUDX0POBs1zBDbwA0sA3gxDkKvPCMHQ-kX1gcAAF9wCAA/language/typescript

Reply:
I expect the input to not change from indentation perspective sinc conditional should have added the indent. Our own formatter does that too and seems very intuitive to me,